fe/src/Component/UserImg/UserImg.styled.tsx

```diff
@@ -2,6 +2,7 @@ import styled, { css } from "styled-components";
 
 type TSizeProp = {
 	size: "small" | "medium";
+	color: string | undefined;
```

Comment:
undefined가 필요하다면 이렇게 작성할 수도 있어용
```suggestion
	color?: string;
```